### PR TITLE
Improved focus styles for storybook examples.

### PR DESCRIPTION
**Pull request by seancurtis**

### CODE REVIEW

File: stories/src/primatives/quote-item.jsx [hunk 1]
```diff
@@ -36,6 +36,9 @@ color: ${colors.black};
   color: ${colors.black};
   text-decoration: none;
 }
+&:focus {
+  outline: 2px solid rebeccapurple;
```
Comment (alexreardon) [inline stories/src/primatives/quote-item.jsx]: love it! However, can you please add a `purple` entry to the `constants > colors` export and use that in all the places?

**Comment by alexreardon** [comment]
You still good to do this one @seancurtis ?

**Comment by seancurtis** [comment]
@alexreardon changes made :)

**Comment by alexreardon** [comment]
Thanks @seancurtis!!!

**Pull request merged**